lib/fs/basicfs_windows.go

```diff
@@ -150,3 +150,31 @@ func (f *BasicFilesystem) Roots() ([]string, error) {
 
 	return drives, nil
 }
+
+func (f *BasicFilesystem) resolveWin83(absPath string) string {
+	if !strings.Contains(absPath, '~') {
+		return absPath
+	}
+	if in, err := windows.UTF16FromString(absPath); err != nil {
+		out = make([]uint16, 2*len(p))
+		n := uint32(len(b))
+		for {
+			n, err = windows.GetLongPathName(&in[0], &out[0], n)
```

Comment:
Where is this coming from?

Reply:
This is a left-over from a zealous fit in which I tried to replace syscall with x/sys wherever possible (due to that weird freeze policy that I admittedly don't understand fully). Then later I decided it was likely a bad idea and definitely not something to be mixed into this 8.3 PR. I can still post a PR with the replacement if you consider it (potentially) worthwhile.